src/main/java/com/ericsson/ei/handlers/MongoDBHandler.java

```diff
@@ -342,4 +343,46 @@ public void dropDatabase(String databaseName) {
         MongoDatabase db = mongoClient.getDatabase(databaseName);
         db.drop();
     }
+    
+    /**
+     * Check if the document exists
+     * @param databaseName
+     * @param collectionName
+     * @param condition is a condition for update documents
```

Comment:
The description of the condition parameter doesn't really make any sense. How about "a condition to find a requested object in the database"? And be consistent in the use of documents vs objects in this method and in the javadoc.

Reply:
done